gateserver/controller_api.py

```diff
@@ -0,0 +1,41 @@
+from .controller_protocol import *
+from . import db
+from . import utils
+
+def get_key_for_mac(mac):
+    """Loads the key for this controller from the DB."""
+    rs = db.exec_sql('SELECT key FROM controller WHERE id = %s',
+                     (utils.bytes2mac(mac),), ret=True)
+    checkmsg(len(rs) == 1, 'unknown controllerID')
+    return rs[0]['key'].tobytes()
```

Comment:
Huh? Co je to ten key, ze ma metodu tobytes?
Jedine co vidim v dokumentacii je "memoryview", ale nie je mi jasne ako sa tu nabral. V kazdom pripade, idiomatickejsie mozno je `bytes(...)`.